pulpcore/pulpcore/etc/pulp/server.yaml

```diff
@@ -68,7 +68,7 @@
 #       formatter: simple
```

Comment:
These lines also need to be removed. They show the defaults.

Reply:
I assumed this was showing users how to set the formatter just like it shows how to configure a syslog handler. I think it would be useful to the discussion to express what the intent for examples in server.yaml are as that would help guide the change needed here.

Reply:
The convention so far is that commented sections are the place for the user to see the exact defaults. Examples are good though, what about moving it to the docs in the logging section?